tests/foreman/ui/test_ui_csv_export.py

```diff
@@ -0,0 +1,732 @@
+"""Test for UI CSV EXPORT
+
+:Requirement: UICSVExport
+
+:CaseLevel: Acceptance
+
+:CaseComponent: UI
+
+:TestType: Functional
+
+:CaseImportance: High
+
+:CaseAutomation: notautomated
+
+:Upstream: No
+"""
+
+from robottelo.decorators import stubbed, tier1
+from robottelo.test import UITestCase
+
+
+class UICSVExportHosts(UITestCase):
```

Comment:
A it is already on `ui` no need the `UI` prefix, call it just `CSVExportHostTestCase` dont forget the `TestCase` suffix.

Reply:
The UI here is part of the feature name, to distinguish it from the CLI CSV export